src/App.scss

```
@@ -1 +1,77 @@
-// not empty
+@import "./variables/variables.scss";
+
+iframe {
+  display: none !important;
+}
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;

```

don't set margins and padding to all components but only to those that need it. 
```suggestion
  box-sizing: border-box;
```